kontrolnaya 1/task3/task3.c

```diff
@@ -0,0 +1,67 @@
+﻿#include <stdio.h>
+#include <time.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define ARRAY_SIZE 100
+#define RANGE 11
+
+const char* deleteRepeating(char* givenString)
+{
+    int stringSize = strlen(givenString);
+    char* resultString = givenString[0];

```

Comment:
```suggestion
    char* resultString = calloc(strlen(givenString) + 1, sizeof(char));
    resultString[0] = givenString[0];
```
Как мы обсуждали после контрольной